tests/e2e/tier-0.fmf

```diff
@@ -0,0 +1,35 @@
+summary: Tier 0 - QM Interconnect
+discover:
+    how: fmf
+    filter: tier:0
+
+provision:
+   how: local
+
+prepare:
+  - name: Prepare Repos
+    how: shell
+    script: |
+        dnf install -y 	dnf-plugin-config-manager epel-release
+        dnf config-manager -y --set-enabled crb
+        dnf -y copr enable rhcontainerbot/qm centos-stream-9
+
+  - name: install repos
+    how: install
+    package:
+      - podman
+
+  - name: Set QM env 
+    how: shell
+    script: |
+        cd tests/e2e
+        bash ./run-test-e2e --skip-tests=yes
```

Comment:
do we intend to skip the tests? Or is this just a temporary thing to get the task successful ?

Reply:
If it's temporary, I'd recommend adding a comment to highlight this.

Reply:
The issue is that,
Once @dougsland submitted the e2e scenario
It is all running under bash scripts

Once running in TMT
All the commands except to tests, are part of  TMT test plan prepare.
At least for now.

The tests script called from run e2e are executed as part of the tests execution
This is the reason for skipping tests in prepare stage,
later on we are running the test script itself with missing env variables